src/styles/utils/_normalizer.scss

```diff
@@ -0,0 +1,11 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
```

Comment:
I think it's better to write from * to body due to the hierarchy
And I consider Roboto have to be in " "